sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -319,4 +323,28 @@ public String getThroughputControlGroupName() {
     public void setThroughputControlGroupName(String throughputControlGroupName) {
         this.throughputControlGroupName = throughputControlGroupName;
     }
+
+    /**
+     * Gets the thresholdForDiagnosticsOnTracer, if latency on CRUD operation is greater than this
+     * diagnostics will be send to open telemetry exporter as events in tracer span of end to end CRUD api.
+     *
+     * Default is 100 ms.
+     *
+     * @return  thresholdForDiagnosticsOnTracerInMS the latency threshold for diagnostics on tracer.
+     */
+    public Duration getThresholdForDiagnosticsOnTracer() {
```

Comment:
please think about if we expect to add more tracer related config (in future), if so maybe we should group all those in a new config called TracerConfig or something like that.

if this is the only config we are planning to add then what we have is fine.

Reply:
As of now i can think of only one config, if in future we have any more, I will create new config file, make sense . Thanks